ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -33,10 +33,9 @@ releases:
         lower: 4.1.0
         upper: 4.999.0
   latest:
-    candidate:
-      product: ocp
-      stream: ci
-      version: "4.13"
```

Comment:
Will this kill our idea of "newest" tests against OCP 4.13?
See tables in https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=StackRox&title=ACS+-+OpenShift+version+support

Reply:
No. See my comment https://github.com/openshift/release/pull/36771#pullrequestreview-1320318054. This PR needs to verify that `4.next` still works after changing the releases stanza. (IIRC the existing values were guesses and when it worked it was :shipit:)

Reply:
Sorry, forgot to circle back on this one. I think changing the release stanza is kind of out of scope for the ticket this PR is associated with so I opened another PR to split this work out (there's some experimentation going on there so don't look at it yet 😅 ): https://github.com/openshift/release/pull/36849

Hopefully, this will make it easier to discuss and verify the tests

When that PR is finalized, I'll remove the release stanza changes from this one

Reply:
This work has been moved to https://github.com/openshift/release/pull/36950